src/components/EditEquipmentListingWizard/EditEquipmentListingWizardTab.js

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { intlShape } from '../../util/reactIntl';
+import routeConfiguration from '../../routeConfiguration';
+import {
+  LISTING_PAGE_PARAM_TYPE_DRAFT,
+  LISTING_PAGE_PARAM_TYPE_NEW,
+  LISTING_PAGE_PARAM_TYPES,
+} from '../../util/urlHelpers';
+import { ensureListing } from '../../util/data';
+import { createResourceLocatorString } from '../../util/routes';
+import {
+  EditListingAvailabilityPanel,
+  EditListingDescriptionPanel,
+  EditListingLocationPanel,
+  EditListingPhotosPanel,
+  EditListingPricingPanel,
+} from '../../components';
+
+import css from './EditEquipmentListingWizard.module.css';
+const LISTING_TYPE = {
+  equipment: 'equipment'
```

Comment:
This Listing_Type const was Duplicated in many files, why don't we declare it in only 1 constant file and export to other files?